2.4/examples/replica/README.md

```diff
@@ -2,20 +2,28 @@
 
 **WARNING:**
 
-**This is only a Proof-Of-Concept example and it is not meant to be used in any
+**This is only a Proof-Of-Concept example and it is not meant to be used in
 production. Use at your own risk.**
 
-## What is MongoDB Replica Set?
+## What is a MongoDB replica set?
 
 A cluster of MongoDB servers that implements master-slave replication and automated failover.
 MongoDB’s recommended replication strategy.
 
 ## Deployment:
 
-To create MongoDB replica set in OpenShift v3, you can use the example template
-included in this repository:
+To create a MongoDB replica set in OpenShift v3, you can use the example template
+included in this repository and:
 
-* `oc create -f ./2.4/examples/replica/mongodb-clustered.json`
+* add a new template to the current project:
+```
+$ oc create -f 2.4/examples/replica/mongodb-clustered.json
```

Comment:
if they follow these instructions they'll still need to take another action to instantiate the template.  that's not very clear from this.

maybe just delete it and only include the new-app instruction.